### DISCUSSION TITLE
Connecting to Fabric with tentant Id

### DISCUSSION THREAD

**Discussion by debugger-mono** [microsoft/SynapseVSCode#25] [Q&A]
We have two instances of the MS Fabric using different tenant IDs but using the same credentials. The plugin only connects to the default instance and provides no way to change the tenant. Is there a no way to connect to different tenant?

**Comment by gbrueckl** [comment]
same issue here - that scenario is actually very common
it would be great if an VSCode config setting exists to set a custom `TenantID`

**Comment by MiSchroe** [comment]
Vote +1
This is needed especially if you work for several different customers and your EntraID is invited as guest user in your customer's tenant.

**Comment by y0m0** [comment]
+1

**Comment by alehpa** [comment]
The same here. We have separate tenant created for our BU only and need to specify MS Fabric instance\tenant

**Comment by djeepy1** [comment]
+1
Must-have for consultants, trainers (and MVPs 😅)

**Reply by teralauritsen** [reply]
+1 
we have accounts in the main tenant and guest accounts in another tenant under the same login ( some AD restriction ),
All of the interesting data is in the non default tenant.

**Comment by TWissing** [comment]
+1

**Comment by sturginho** [comment]
Same issue here, the workspace I need to access is not on my default tenant, but there is no way for me to select a different tenant

**Comment by lindacmsheard** [comment]
+1!

**Comment by okayakdeniz** [comment]
+1 Also, Sign-in method must be change.

**Comment by celindho** [comment]
+1

**Comment by famkevanree** [comment]
+1

**Comment by qixwang** [comment]
supported by the release of 1.18.1. close this issue.

**Reply by gbrueckl** [reply]
> supported by the release of 1.18.1. close this issue.

awesome, thanks!
